pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -221,6 +221,7 @@ func (h *heartbeatSender) startHeartbeatLoop(ctx context.Context) {
 				case frontier := <-h.frontierUpdates:
 					h.frontier.Forward(frontier)
 				}
+				metrics.FrontierLagSeconds.Update(timeutil.Since(h.frontier.GoTime()).Seconds())
```

Comment:
Do we think it is better to put this here rather than in `maybeUpdatePartitionProgress`? If so, some commentary on why might be good.

Reply:
I think your suggestion would be better, thanks! done. PTAL.